bin/node/runtime/src/lib.rs

```diff
@@ -66,6 +66,7 @@ use pallet_contracts_rpc_runtime_api::ContractExecResult;
 use pallet_session::{historical as pallet_session_historical};
 use sp_inherents::{InherentData, CheckInherentsResult};
 use static_assertions::const_assert;
+pub use pallet_cere_ddc;
```

Comment:
Why it's `pub`?

Reply:
For other module can call. Like "pub use pallet_transaction_payment::{Multiplier, TargetedFeeAdjustment};"